recipes/pycognito/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "pycognito" %}
+{% set version = "2021.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3a496364b697d744b37187d0ca3d950abc7540653455240477e4a41c3ba887a1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Looks like pure Python so it could be noarch Python (reduces the number of builds):
```suggestion
build:
  number: 0
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"
```